### PR TITLE
List Vertcoin (VTC)

### PR DESCRIPTION
**Pull request by cruelnovo**
Author: Matt Cummings <matt@gocummings.com>
Date: Tue Jun 18 20:36:00 2019 -0500

     List Vertcoin (VTC)

- Official project URL: https://vertcoin.org/
- Official block explorer URL: https://insight.vertcoin.org/

### CODE REVIEW

**Comment by cruelnovo** [comment]
- Official project URL: https://verticoin.org/
- Official block explorer URL: https://insight.vertcoin.org/

File: assets/src/main/java/bisq/asset/coins/Vertcoin.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+package bisq.asset.coins;
+
+import bisq.asset.Base58BitcoinAddressValidator;
+import bisq.asset.Coin;
+import bisq.asset.NetworkParametersAdapter;
+
+public class Vertcoin extends Coin {
+    public Vertcoin() {
+        super("Vertcoin", "VTC", new Base58BitcoinAddressValidator(new VertcoinMainNetParams()), Network.MAINNET);
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/Vertcoin.java]: `Network.MAINNET` is default so remove it.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments.

Apart from that code looks good.

**Comment by blabno** [comment]
Squash the commits into a single on, keeping the message from the first commit.

**Review by blabno — APPROVED** [review]
ACK

**Pull request merged**